src/Analysis/Ast/Impl/Indexing/DirectoryFileReader.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Extensions.FileSystemGlobbing;
+using Microsoft.Extensions.FileSystemGlobbing.Abstractions;
+using Microsoft.Python.Core;
+using Microsoft.Python.Core.IO;
+using System;
```

Comment:
Sorting - system first. There is a setting in VS (Tools | Options | Text Editor | C#) or just import editor settings file from the repo.